formats/detailed_guide/publisher/links.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "topics",
+    "lead_organisations"
+  ],
+  "properties": {
+    "related_guides": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_format_provider_organisation": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "related_mainstream": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "additional_related_mainstream": {
+      "$ref": "#/definitions/guid_list"
+    },
+    "alternative_scotland_url": {
```

Comment:
Spelunking (yes, I do like this word and I will use it a lot) reveals they are external indeed URLs, and as such should be part of `details` and not `links`.

E.g., https://www.gov.uk/guidance/storing-oil-at-a-home-or-business has excluded nations:

```ruby
> d = Document.find_by_slug("storing-oil-at-a-home-or-business")
> dg = d.latest_edition
> dg.nation_inapplicabilities.last
<NationInapplicability id: 122149, nation_id: 4, edition_id: 574326, created_at: "2015-02-25 17:22:38", updated_at: "2015-12-16 17:13:00", alternative_url: "https://www.doeni.gov.uk/publications/control-poll...">
```

The `alternative_url` is fully qualified, so it's external.

cc @boffbowsh